Essentials/src/main/java/com/earth2me/essentials/PlayerList.java

```diff
@@ -37,7 +37,9 @@ public static String listUsers(final IEssentials ess, final List<User> users, fi
             }
             user.setDisplayNick();
             groupString.append(user.getDisplayName());
-            if (ess.getSettings().realNamesOnList() && !ChatColor.stripColor(user.getDisplayName()).equals(user.getName())) {
+
+            final String strippedNick = FormatUtil.stripEssentialsFormat(user.getNickname());
```

Comment:
Shouldn't `stripFormat` be used since `stripEssentialsFormat` does untranslated color codes afaik?
```suggestion
            final String strippedNick = FormatUtil.stripFormat(user.getNickname());
```

Reply:
I really just assumed that Essentials stored the nicknames in Essentials format without checking. I am a clown.